bot.py

```diff
@@ -193,13 +193,23 @@ def pr_url(self, version):
             await self.say("Want to start a release? hhhmmmmm")
 
 
+class BotException(Exception):
+    """Exception raised for invalid input.
```

Comment:
This can just be 

    class BotException(Exception):
        """Exception raised for invalid input."""

`Exception.__init__` will take a message already so we don't need to override it